rubrics/tool/src/main/frontend/webcomponents/js/sakai-rubric.js

```diff
@@ -162,6 +200,94 @@ export class SakaiRubric extends RubricsElement {
     this.updateRubric();
   }
 
+  handleCriterionCreated(e) {
+    var data = e.detail;
+    this.rubric.criterions.push(data.criterion);
+  }
+
+  handleSaveWeights(e) {
+    var _this = this;
+    $(`[rubric-id=${this.rubric.id}]`).find('.save-weights').prop('disabled', true);
```

Comment:
I'd prefer you to use vanilla js here. We're trying to not use jquery unless it offers a massive advantage.

Reply:
Done